src/csharp/Microsoft.Spark/Services/ConfigurationService.cs

```diff
@@ -33,6 +33,18 @@ internal sealed class ConfigurationService : IConfigurationService
 
         private string _workerPath;
 
+        /// <summary>
+        /// How often to run GC on JVM ThreadPool threads.
```

Comment:
You can mention default value here instead.

Reply:
Added.